### PR TITLE
Closed questions

### PR DESCRIPTION
**Pull request by Bruce-Connor**
Indicate closed questions. Builds on `delete-command`

### CODE REVIEW

File: sx-question-print.el [hunk 1]
```diff
@@ -218,6 +230,15 @@ QUESTION must be a data structure returned by `json-read'."
   (goto-char (point-min))
   (sx-question-mode-next-section))
 
+(defun sx-question-mode--print-close-reason (reason date)
+  "Print a header explaining REASON and DATE.
+DATE is an integer."
+  (insert "\n    "
+          (propertize
+           (format "Closed %s ago because %s" reason (sx-time-since date))
```
Comment (vermiculus) [inline sx-question-print.el]: shouldn't this be 
```lisp
(format "Closed %s ago because %s" (sx-time-since date) reason)
```
?

Comment (Bruce-Connor) [inline sx-question-print.el]: Oops. Yes, it should.

**Comment by Bruce-Connor** [comment]
Here's a [closed question](http://emacs.stackexchange.com/q/9618/50) for reference on how this looks.

**Comment by vermiculus** [comment]
:+1:

**Pull request merged**